keywords/couchbaseserver.py

```diff
@@ -184,6 +184,18 @@ def delete_buckets(self):
         if len(bucket_names) != 0:
             raise CBServerError("Failed to delete all of the server buckets!")
 
+        # verify all indexes are deleted
+        count = 0
+        index_url = self.url.replace("8091", "9102")
+        while count < 5:
+            resp = self._session.get("{}/getIndexStatus".format(index_url))
```

Comment:
When I was adding retry I have explore this api  "https://pypi.org/project/retry/"  and it is very good. I couldn't use it since my use case are more than a api.

Reply:
sounds great. by using this retry api, do we need to add this to the requirement lists to ensure the python package gets installed?

Reply:
cool, yes it needs new package. Let's try this out when we do it next time